scim2-sdk-common/src/main/java/com/unboundid/scim2/common/GenericScimResource.java

```diff
@@ -688,36 +648,36 @@ public GenericScimResource addStringValues(@NotNull final String path,
   }
 
   /**
-   * Adds String values to an array node.  If no array node exists at the
-   * specified path, a new array node will be created.
-   *   <p>
-   * For example:
-   * In a GenericScimResource (gsr) representing the following resource:
+   * Adds new values to a multi-valued attribute of strings in a generic SCIM
+   * resource. If no ArrayNode exists at the specified path, a new ArrayNode
+   * will be created.
+   * <br><br>
+   *
+   * For example, consider the following GenericScimResource:
+   * <pre>
+   *   {
+   *     "starRealmsCards": [ "Trade Bot", "Cutter" ]
+   *   }
+   * </pre>
+   *
+   * To add new values, use one of the following:
    * <pre><code>
-   * {
-   *   "path1":[ "stringValue1", "stringValue2" ]
-   * }
-   * </code></pre>
-   *   <p>
-   *   gsr.addStringValues(Path.fromString("path1"), path1values)
-   *   where path1Value is a List of String.
-   *   Would add each of the items in the path1values list to the
-   *   "path1" list in the generic SCIM resource
-   *   <p>
+   *   gsr.addStringValues("starRealmsCards", "BattleCruiser);
+   *
+   *   // This method is useful when the new values are already in a list.
+   *   gsr.addStringValues("starRealmsCards", List.of("BattleCruiser"));
    *
-   *   gsr.addStringValues(Path.fromString("path2"), path2values)
-   *   where path2values is a List of String.
-   *   Would create a new array called "path2"
-   *   <p>
+   *   // Creates a new array of values on the resource.
```

Comment:
It might be worth expanding this comment (and the matching ones for the other data types) to something like 
```
// Creates a new array of values called "sevenWondersCard" on the resource.
```

Reply:
I considered this, but this makes the sentence a bit wordier and I was worried that sentences like that might require a re-read to understand. My hope is that the code makes it clear what the key name will be in the JSON.